index.html

```diff
@@ -58,155 +74,175 @@ <h2 class="sub-title">I am a Softwaer Developer</h2>
-                
-            </ul>
-        </div>
-        <a href="#" class="pro-card-button">See Project</a>
-    
-       </div>
-  </section>
-   <!--professional part start 1-->
-  <section class="professional-card">
-    <div class="card2">
-        <div class="card-title">Profesional Art<br> Printing Data</div>
-        <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
-        <div class="card-pro-buttons">
-            <ul class="pro-buttons">
-                <li><a href="#">html</a></li>
-                <li><a href="#">bootstrap</a></li>
-                <li><a href="#">Ruby</a></li>
-                
-            </ul>
-        </div>
-        <a href="#" class="pro-card-button">See Project</a>
-    
-       </div>
-  </section>
-   <!--professional part start 1-->
-  <section class="professional-card">
-    <div class="card2">
-        <div class="card-title">Profesional Art<br> Printing Data</div>
-        <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
-        <div class="card-pro-buttons">
-            <ul class="pro-buttons">
-                <li><a href="#">html</a></li>
-                <li><a href="#">bootstrap</a></li>
-                <li><a href="#">Ruby</a></li>
-                
-            </ul>
-        </div>
-        <a href="#" class="pro-card-button">See Project</a>
-    
-       </div>
-  </section>
-   <!--professional part start 1-->
-  <section class="professional-card">
-    <div class="card2">
-        <div class="card-title">Profesional Art<br> Printing Data</div>
-        <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
-        <div class="card-pro-buttons">
-            <ul class="pro-buttons">
-                <li><a href="#">html</a></li>
-                <li><a href="#">bootstrap</a></li>
-                <li><a href="#">Ruby</a></li>
-                
-            </ul>
-        </div>
-        <a href="#" class="pro-card-button">See Project</a>
-    
-       </div>
-  </section>
-   <!--professional part start 1-->
-  <section class="professional-card">
-    <div class="card2">
-        <div class="card-title">Profesional Art<br> Printing Data</div>
-        <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
-        <div class="card-pro-buttons">
-            <ul class="pro-buttons">
-                <li><a href="#">html</a></li>
-                <li><a href="#">bootstrap</a></li>
-                <li><a href="#">Ruby</a></li>
-                
-            </ul>
-        </div>
-        <a href="#" class="pro-card-button">See Project</a>
+<section class="profession-group">
```

Comment:
- [ ] Kindly add the right spacing between the content of the card as shown in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/215283650-5cbcb92f-c044-40d1-bdd9-904b7412bb48.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/215283662-e6d05bae-0ccd-407a-b818-5b867e249523.png)